README.mkd

```diff
@@ -93,7 +93,17 @@ If you experience any troubles, try running:
 thumbor-doctor
 ```
 
-If you still need help, please [raise an issue](https://github.com/thumbor/thumbor/issues).
+If you have a `thumbor.conf` file, you can use that to help thumbor-doctor:
+
+```bash
+thumbor-doctor -c thumbor.conf
+```
+
+If you still need help, please [raise an issue](https://github.com/thumbor/thumbor/issues). Remember to send your `thumbor-doctor` output in the issue:
```

Comment:
This should be documented in the github issue template instead.

Reply:
Maybe in both?